apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -176,6 +176,14 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
             },
         },
 
+        primaryFile: {
```

Comment:
Why it’s primary? We will have a secondary file?

Reply:
probably in future we can have other files related to the billingReceipt, but one of them going to be "primary" one.

Reply:
There will be no other fields here. Better, to rename it

Reply:
renamed to "file"